src/Events/Listeners/CatchSluggableRoutes.php

```diff
@@ -69,9 +70,10 @@ protected function startSession(Request $request)
     public function getSession(Request $request)
     {
         return tap($this->sessionManager->driver(), function ($session) use ($request) {
-            if ($encryptedSessionId = $request->cookies->get($session->getName())) {
-                $sessionId = $this->encrypter->decrypt($encryptedSessionId);
+            $isSerialized = EncryptCookies::serialized($session->getName());
```

Comment:
@ryzr can you verify that tabs or spaces would be better here please?

Reply:
<img width="322" alt="Screen Shot 2022-12-12 at 10 10 03 am" src="https://user-images.githubusercontent.com/93168303/206935922-d280ce8e-caa0-495b-b43d-1dfd67286fcf.png">

I recommend something like this.

Reply:
Should probably also add a test

```php
<?php

namespace Tests;

use Illuminate\Foundation\Testing\DatabaseMigrations;
use Illuminate\Foundation\Testing\TestCase;

class FizzBuzzTest extends TestCase
{
    use CreatesApplication, DatabaseMigrations;

    public function testItPerformsFizzBuzzCorrectly()
    {
        $this->runDatabaseMigrations();
        $this->seed();

        $this->assertEquals("12Fizz4BuzzFizz78FizzBuzz11Fizz1314FizzBuzz1617Fizz19BuzzFizz2223FizzBuzz26Fizz2829FizzBuzz3132Fizz34BuzzFizz3738FizzBuzz41Fizz4344FizzBuzz4647Fizz49BuzzFizz5253FizzBuzz56Fizz5859FizzBuzz6162Fizz64BuzzFizz6768FizzBuzz71Fizz7374FizzBuzz7677Fizz79BuzzFizz8283FizzBuzz86Fizz8889FizzBuzz9192Fizz94BuzzFizz9798FizzBuzz", logFizzBuzz(), 'Bad Fizz Buzz');
    }
}

```